newton/_src/solvers/mujoco/solver_mujoco.py

```
@@ -226,6 +286,10 @@ def import_mujoco(cls):
                 raise ImportError(
                     "MuJoCo backend not installed. Please refer to https://github.com/google-deepmind/mujoco_warp for installation instructions."
                 ) from e
+        try:
+            _warn_if_mujoco_versions_mismatch(cls._mujoco, cls._mujoco_warp)
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)